src/main/flight/pid.h

```diff
@@ -113,6 +113,13 @@ PG_DECLARE_ARRAY(pidProfile_t, MAX_PROFILE_COUNT, pidProfiles);
 
 typedef struct pidConfig_s {
     uint8_t pid_process_denom;              // Processing denominator for PID controller vs gyro sampling rate
+#ifdef USE_RUNAWAY_TAKEOFF
```

Comment:
This cannot be conditional. The goal is to use parameter groups to programmatically generate interfaces (to be used over MSP). This means that every version of a parameter group has to be identical for all targets that support it.